### PR TITLE
Implement grid-style model selection in Results Tuner

### PR DESCRIPTION
**Pull request by CFIALeronB**
- Added a grid layout to display classification models, allowing users to choose from a list of models.
- Integrated real and static data sources for model details.
- Implemented a toggle functionality for switching between live model data and static test data for development ease.
- The selected model is now globally accessible via `selectedModel` state.
- Introduced a `BugReportIcon` (or your chosen icon) for toggling between real and test data, symbolizing debugging and development settings.
- Ensured robust error handling and state management for asynchronous data fetching from the backend (if applicable) or static data.

ModelSelectionScreenshot1

ModelSelectionScreenshot2

#28 #93

### CODE REVIEW

**Review by rngadam — APPROVED** [review]

**Comment by rngadam** [comment]
@ThomasCardin a case where the PR is done within the user account instead of the organization, we get:

```
Error: buildx failed with: ERROR: invalid tag "ghcr.io/ai-cfia/nachet-frontend:CFIALeronB/issue28-ModelSelection": invalid reference format
```
probably need to strip whatever is before /

**Comment by rngadam** [comment]
@CFIALeronB you're using merging to catch up to changes in the main branch, but we want to do rebase to main to keep a clean history.

see: https://github.com/ai-cfia/.github/pull/5

please rebase. you also have a check failing.

File: src/_versions.ts [hunk 1]
```diff
@@ -11,8 +11,8 @@ export interface TsAppVersion {
 export const versions: TsAppVersion = {
     version: '0.2.0',
```
Comment (rngadam) [inline src/_versions.ts:12]: MINOR version should be updated in package.json as this introduces new functionality.

https://semver.org/

**Review by rngadam — CHANGES_REQUESTED** [review]
fix check, version and rebase on main

File: src/_versions.ts (hunk 1 above)
Comment (rngadam) [reply src/_versions.ts:12]: @CFIALeronB don't see a reply to this one.

Comment (CFIALeronB) [reply src/_versions.ts]: @rngadam done!

**Comment by rngadam** [comment]
Created https://github.com/ai-cfia/github-workflows/issues/94 

but @CFIALeronB can move this branch to the ai-cfia/nachet-frontend instead of a fork into your own repo to get the build to complete 100% in the meantime?

**Pull request closed without merging**